test/unit/messaging/messaging.spec.ts

```diff
@@ -3534,42 +3545,60 @@ describe('Messaging', () => {
       },
     ];
 
+    const GAPIC_RESPONSE = [
+      // An example IMessage that is normally returned by the generated client
+      // when sendMessage() is ran successfully
+      { name: 'projects/projec_id/messages/message_id' }
+    ];
+
     validMessages.forEach((config) => {
       it(`should serialize well-formed Message: ${config.label}`, () => {
         // Wait for the initial getToken() call to complete before stubbing https.request.
         return mockApp.INTERNAL.getToken()
           .then(() => {
-            const resp = utils.responseFrom({ message: 'test' });
-            httpsRequestStub = sinon.stub(HttpClient.prototype, 'send').resolves(resp);
+            httpsRequestStub = sinon.stub(FcmServiceClient.prototype, 'sendMessage').resolves(GAPIC_RESPONSE);
```

Comment:
This test can be further simplified now. The `mockApp.INTERNAL.getToken()` part should no longer be necessary.

Reply:
Shouldn't we use `generatedClientStub` here?

Reply:
Yes! Thanks for catching that.